### PR TITLE
Set pointer-events to none on icons

### PR DESCRIPTION
**Pull request by JCB-K**
We have some strange error where icons inside react-router <Link /> components are not clickable.
By setting pointer-events to none the event bubbles up to the parent.

Also moved some boilerplate from index.js to icon/index.js.

### CODE REVIEW

**Comment by bstream** [comment]
Rather set this in defaultProps for icon.jsx, then it will propagate to all icons :)

**Comment by JCB-K** [comment]
merge #218 before reviewing this

File: test/components/icon/icon.test.js [hunk 1]
```diff
@@ -22,6 +22,7 @@ describe('<Icon />', () => {
 
   it('should be able to override stroke', () => {
     const color = 'tomato';
+    // Icon.forEach(i => console.log(i.style))
```
Comment (bstream) [inline test/components/icon/icon.test.js]: Ugh!

Comment (JCB-K) [inline test/components/icon/icon.test.js]: check again :smirk:

**Pull request merged**